lib/posix-user/exportsyms.uk

```diff
@@ -64,3 +64,6 @@ getgrnam_r
 setgrent
 endgrent
 getgrent
+setresuid
```

Comment:
Just a nitpick. This is redefined ([line 50](https://github.com/unikraft/unikraft/blob/af34c19cf9fa3c8ab8888c91a5e62b05e0a66315/lib/posix-user/exportsyms.uk#L50)).